tensorboard/plugins/interactive_inference/utils/inference_utils.py

```diff
@@ -230,7 +230,10 @@ def __init__(self, inference_address, model_name, model_type, model_version,
 
 def ensure_not_binary(value):
   """Return non-binary version of value."""
-  return value.decode() if isinstance(value, binary_type) else value
+  try:
+    return value.decode() if isinstance(value, binary_type) else value
+  except UnicodeDecodeError:
```

Comment:
when does this happen? can you put a short comment here?

Reply:
done